command/s3_put_test.go

```diff
@@ -269,9 +270,10 @@ func TestExpandS3PutParams(t *testing.T) {
 			So(cmd.ContentType, ShouldEqual, "ct")
 			So(cmd.ResourceDisplayName, ShouldEqual, "file")
 			So(cmd.Visibility, ShouldEqual, "private")
-			So(cmd.workDir, ShouldEqual, "working_directory")
 			So(cmd.Optional, ShouldEqual, "true")
 
+			// EVG-7226 Since LocalFile is an absolute path, workDir should be empty
+			So(cmd.workDir, ShouldEqual, "")
```

Comment:
It took some time for me to understand that `${workdir}` being expanded to the empty string leaves an absolute path an Linux.
Will this pass on Windows?